sdk/storage/azdatalake/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+## 1.1.2 (Unreleased)
+
+### Features Added
+* Append with acquire, release lease and renewal of lease.
```

Comment:
this needs to be more explanatory in terms of feature.

Reply:
made changelog feature name more intuitive